packages/client-sdk-nodejs/src/config/retry/default-eligibility-strategy.ts

```diff
@@ -31,32 +31,57 @@ const retryableGrpcStatusCodes: Array<Status> = [
 ];
 
 const retryableRequestTypes: Array<string> = [
```

Comment:
Compare to the [Go SDK version](https://github.com/momentohq/client-sdk-go/blob/9ebbbe63aefbfdfee3c0fcbe574973980838b54b/internal/retry/eligibility_strategy.go#L16).